frontend/src/mocks/msw/handlers/boardsLists/boardsListsHandler.js

```diff
@@ -1,8 +1,15 @@
 import { rest } from 'msw';
-import { boardsListsResponse } from './boardsListsResponse';
+import { boardsListResponse } from './BoardsListResponse';
 
-const boardsListsHandler = rest.get('/boards', (req, res, ctx) => {
-  return res(ctx.status(200), ctx.json(boardsListsResponse));
+const getAllBoardsHandler = rest.get('/boards', (req, res, ctx) => {
+  return res(ctx.status(200), ctx.json(boardsListResponse));
 });
 
```

Comment:
```suggestion
const getAnnounceCementListResponse = rest.get('/boards/:id/announcements', (req, res, ctx) => {
  const { id } = req.params;
  return res(ctx.status(200), ctx.json(boardsListResponse[id].announcements));
});

const postAnnounceCementListResponse = rest.post('/boards/:id/announcements', (req, res, ctx) => {
  const { id } = req.params;
  boardsListResponse[id].announcements.push(req.body);
  return res(ctx.status(200), ctx.json(boardsListResponse[id].announcements));
});

```

Reply:
Tak by wyglądało to co mówiłem, że bym przeniósł

Reply:
I w ogóle zmienił bym nazwę folderu i plików na po prostu boards/boardsHandler/boardsResponse xD